v2/concordium/types.proto

```diff
@@ -503,6 +503,58 @@ message Cooldown {
   CooldownStatus status = 3;
 }
 
+// The unique id of a token. The identifier is the position in the token table.
+message TokenId {
+  uint64 value = 1;
+}
+
+// The unique identifier of a token module.
+message TokenModuleId {
+  uint64 value = 1;
+}
+
+// PLT amount representation.
+message TokenAmount {
+  uint64 amount = 1;
```

Comment:
I don't know how important this is to the use cases, but on a number of other chains they use `u256` to represent a token amount.

Reply:
Afaik, there is not built-in protobuf `u256` type. Might depend a bit on the representation we are using for the amount. There is also the google money type in protobuf which is `uint64` units + `uint32` nano fractional parts.

Reply:
u256 seems way overkill and I imagine u64 is plenty, but just wanted to bring this up for discussion